### PR TITLE
Release v1.2 of NNCF for PyTorch

### PR DESCRIPTION
**Pull request by vshampor**

### CODE REVIEW

File: pytorch_toolkit/nncf/examples/object_detection/configs/ssd_mobilenet_voc_rb_sparsity_int8.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
   "model": "ssd_mobilenet",
-  "weights": "<WEIGHTS_PATH>",
   "batchnorm": true,
-  "input_sample_size": [8, 3, 300, 300],
+  "input_info": {
+    "sample_size": [8, 3, 300, 300]
```
Comment (AlexKoff88) [inline pytorch_toolkit/nncf/examples/object_detection/configs/ssd_mobilenet_voc_rb_sparsity_int8.json]: I assume this is not a problem but do we really need batch 8 in this config?

File: pytorch_toolkit/nncf/README.md [hunk 2]
```diff
@@ -1,73 +1,82 @@
-# Neural Network Compression Framework (NNCF)
+# Neural Network Compression Framework  (NNCF)
```
Comment (AlexanderDokuchaev) [inline pytorch_toolkit/nncf/README.md]: This PR have reverted all changes in .md files from https://github.com/opencv/openvino_training_extensions/pull/211

Comment (AlexanderDokuchaev) [inline pytorch_toolkit/nncf/README.md]: This PR have reverted all changes in .md files from https://github.com/opencv/openvino_training_extensions/pull/211

Comment (vshampor) [reply pytorch_toolkit/nncf/README.md]: Will rectify.

Comment (vshampor) [reply pytorch_toolkit/nncf/README.md]: Done

**Comment by AlexanderDokuchaev** [comment]
Jenkins please retry a build

**Comment by AlexanderDokuchaev** [comment]
run tests

**Comment by AlexanderDokuchaev** [comment]
@vshampor Do you have access to [Jenkins](http://134.191.240.124/job/nncf/job/precommit/job/nncf-tests/105/)? 
10 tests failed.

**Comment by vshampor** [comment]
@AlexanderDokuchaev , the checks have passed. Propose to merge this.

**Pull request merged**